mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOpsTypes.h

```diff
@@ -53,6 +53,8 @@ enum class AccessAddrSpace : int {
   ExtIntelHost = 6
 };
 
+enum class DecoratedAccess : int { No = 0, Yes = 1, Legacy = 2 };
```

Comment:
I'm more inclined to `AccessDecorated`, as that's more consistent with the current naming we're using, but I'm  also fine with this.

Reply:
I think putting the adjective before the noun reads better here, sorry, keeping it like that, hope OK